AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -279,6 +228,80 @@ std::map< int, std::vector<int> > RayTraceCorrelator::SetupPairs(
     return pairs;
 }
 
+std::pair< 
+    std::vector< std::vector< std::vector< int > > >,
+    std::vector< std::vector< std::vector< double > > > > RayTraceCorrelator::GetArrivalDelays(std::map<int, std::vector<int > > pairs){
+
+    std::vector< std::vector< std::vector< double > > > delays;
+    std::vector< std::vector< std::vector< int > > > bins;
+
+    for(int solNum=0; solNum<2; solNum++){
+
+        std::vector< std::vector<double> > delays_this_sol;
+        std::vector< std::vector<int> > bins_this_sol;
+
+        for(auto iter = pairs.begin(); iter != pairs.end(); ++iter){
+            int pairNum = iter->first;
+            int ant1 = iter->second[0];
+            int ant2 = iter->second[1];
+
+            std::vector<double> this_delays;
+            std::vector<int> this_global_bins;
+
+            for(int phiBin = dummyMap->GetXaxis()->GetFirst(); phiBin <= dummyMap->GetXaxis()->GetLast(); phiBin++){
+                for(int thetaBin = dummyMap->GetYaxis()->GetFirst(); thetaBin <= dummyMap->GetYaxis()->GetLast(); thetaBin++){
+
+                    int globalBin  = dummyMap->GetBin(phiBin, thetaBin);
+                    double arrival_time1 = arrvialTimes_.at(solNum).at(ant1).GetBinContent(globalBin);
```

Comment:
is this typo okay `arrvialTimes`. Same in the line below

Reply:
I made the typo everywhere, so it turned out not to matter. But I fixed it! Thanks for catching it.